main.go

```diff
@@ -23,11 +23,13 @@ import (
 	"github.com/openshift/sippy/pkg/buganalysis"
 	"github.com/openshift/sippy/pkg/db"
 	"github.com/openshift/sippy/pkg/perfscaleanalysis"
+	prowanalysis "github.com/openshift/sippy/pkg/prowloader"
```

Comment:
Could we do either a prowanalysis as actual package name, or use prowloader as the import name? May as well name it what we'd like to call it.

Reply:
Yea this is just an artifact of me not being able to decide on a name. I'm still in the process of trying to organize the code so ignore names and paths for now :)